src/popup/components/buttons/BtnSubheader.vue

```diff
@@ -53,7 +53,7 @@ export default defineComponent({
   display: flex;
   align-items: center;
   width: 100%;
-  padding: 20px 12px;
+  padding: 20px 16px;
```

Comment:
This component is used in 5 places. Are your sure it should be updated? Is this change aligned with the design/designer?

Reply:
> This component is used in 5 places. Are your sure it should be updated? Is this change aligned with the design/designer?

Yes, this was communicated and agreed with the designer.